test/unit/transloadit/test_assembly.rb

```diff
@@ -97,14 +97,47 @@
         Transloadit::Assembly.new(
           @transloadit,
           :fields => {:tag => 'ninja-cat'}
-        ).submit!
+        ).create!
 
         assert_requested(:post, 'api2.transloadit.com/assemblies') do |req|
           values = values_from_post_body(req.body)
           values['tag'].must_equal 'ninja-cat'
           MultiJson.load(values['params'])['fields']['tag'].must_equal 'ninja-cat'
         end
       end
+
+      it 'must allow steps through create! method' do
```

Comment:
A missing "the". Let's use correct grammar here. "Must allow steps through the create! method".